src/components/Card/CardBody.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Context } from './Context';
+
+const CardBody = ({ children, additionalClasses = [] }) => {
+    const context = useContext(Context);
+    const directionClass = context && context.vertical
+        ? 'flex flex-col items-start justify-between basis-full' : '';
+    let basisClass = '';
+    if (context && context.vertical && context.proportionsForVertical && context.proportionsForVertical.body) {
+        const { body, breakpoint } = context.proportionsForVertical;
+        basisClass = ` ${breakpoint ?? 'md'}:basis-${body}`;
+    }
+    useEffect(() => {
+        if (context && context.vertical && context.proportionsForVertical && context.proportionsForVertical.img) {
```

Comment:
Shouldn't be the same values?
![card-component-by-CiotkaCierpienia-·-Pull-Request-5-·-flotiq-flotiq-components-react](https://user-images.githubusercontent.com/645716/150518314-7569d83b-2f99-40df-abc6-a060d7f718f3.png)